Utils.py

```diff
@@ -317,7 +317,14 @@ def get_default_options() -> OptionsType:
         },
         "wargroove_options": {
             "root_directory": "C:/Program Files (x86)/Steam/steamapps/common/Wargroove"
-        }
+        },
+        "adventure_options": {
+            "rom_file": "ADVNTURE.BIN",
+            "display_msgs": True,
+            "rom_start": True,
+            "rom_args": "",
+            "display_msgs": True
```

Comment:
duplicate key

Reply:
Oh, yeah, I thought I had removed from Utils.py at some point and was putting it back, but apparently I had only removed it from my xml.  Kept changing my mind whether to have it be an option.